src/main/java/com/brandwatch/internship/fetchandstoretweets/services/FetchAndStoreService.java

```diff
@@ -0,0 +1,67 @@
+package com.brandwatch.internship.fetchandstoretweets.services;
+
+import com.brandwatch.internship.fetchandstoretweets.entities.Mention;
+import com.brandwatch.internship.fetchandstoretweets.entities.Query;
+import com.brandwatch.internship.fetchandstoretweets.repositories.MentionsRepository;
+import com.brandwatch.internship.fetchandstoretweets.repositories.QueryRepository;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.social.twitter.api.SearchResults;
+import org.springframework.social.twitter.api.Tweet;
+import org.springframework.social.twitter.api.Twitter;
+import org.springframework.stereotype.Service;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
+
+@Service
+public class FetchAndStoreService {
```

Comment:
Usually service class names are nouns in format `<Entity this service uses the most>Service`. It will be more conventional to call it `MentionService` or `TweetsService`.